scripts/ansible

```diff
@@ -185,15 +185,28 @@ $APT_PATH/apt -y install python3-pip
 # https://github.com/iiab/iiab/pull/3022
 pip3 config --global set global.no-cache-dir false
```

Comment:
I should have mentioned earlier: this line also fails every time (`pip3` is not installed).

Reply:
corrected